conf/modules.config

```diff
@@ -139,8 +139,8 @@ process {
                 "${wfmash_sparse_map_cmd}",
                 params.wfmash_temp_dir         ? "-B ${wfmash_temp_dir}"             : "",
                 "-m",
-                "-2 ${params.wfmash_hg_filter_ani_diff}",
-                "--lower-triangular",
+//                "-2 ${params.wfmash_hg_filter_ani_diff}",
+//                "--lower-triangular",
```

Comment:
```suggestion
```

Reply:
I want to keep this for future releases of `wfmash`. Would that be fine?